package.json

```diff
@@ -157,6 +157,7 @@
 		"enzyme": "3.11.0",
 		"equivalent-key-map": "0.2.2",
 		"eslint-plugin-eslint-comments": "3.1.2",
+		"eslint-plugin-import": "2.22.1",
```

Comment:
We use it in the whole project again 👍

You probably should rebase this branch again, remove the changes from the lock file and run `npm i`. There should be only one line updated.

Reply:
Makes sense! Will try!